radix-engine/src/kernel/kernel_api.rs

```diff
@@ -202,26 +202,41 @@ pub struct SystemState<'a, M: KernelCallbackObject> {
     pub caller_call_frame: &'a M::CallFrameData,
 }
 
-/// Internal API for kernel modules.
-/// No kernel state changes are expected as of a result of invoking such APIs, except updating returned references.
+/// Internal API for system modules only.
+///
+/// TODO: Do not use uncosted API within protocol
```

Comment:
Is this TODO meant to be done in this PR? Or in future protocol updates?

Reply:
Future protocol updates. They're very cheap (reading a variable from memory), so there is no urgency in fixing them.